test/test_automl/test_automl.py

```diff
@@ -114,6 +115,33 @@ def test_fit(self):
         self._tearDown(backend_api.temporary_directory)
         self._tearDown(backend_api.output_directory)
 
+    def test_delete_non_winning_models(self):
+        backend_api = self._create_backend(
+            'test_delete', delete_tmp_folder_after_terminate=False)
+
+        X_train, Y_train, _, _ = putil.get_dataset('iris')
+        automl = autosklearn.automl.AutoML(
+            backend_api,
+            time_left_for_this_task=20,
+            per_run_time_limit=5,
+            ensemble_nbest=3,
+        )
+        automl.fit(
+            X_train, Y_train, metric=accuracy, task=MULTICLASS_CLASSIFICATION,
+        )
+
+        model_files = glob.glob(os.path.join(
+            backend_api.temporary_directory, '.auto-sklearn', 'models', '*.model'))
+
+        # Assert that the files of the models used by the ensemble are available
+        ensemble_members = set([idx[1] for idx in automl.ensemble_.identifiers_])
```

Comment:
Could you please rename the variables so that they indicate that these are the indices? This would make the code a bit easier to understand.

Reply:
Ok